mod/state-transition/pkg/core/state_processor_genesis.go

```diff
@@ -166,5 +166,5 @@ func (sp *StateProcessor[
 		return nil, err
 	}
 
-	return sp.processSyncCommitteeUpdates(st)
```

Comment:
`processSyncCommitteeUpdates` does not really make sense to me.
It seems to me we're trying to squeeze a pure BeaconKit operation (updating CometBFT val set) into an Eth 2.0 one